app_dart/lib/src/model/firestore/task.dart

```diff
@@ -285,21 +289,39 @@ List<Task> targetsToTaskDocuments(Commit commit, List<Target> targets) {
 }
 
 /// Generates task document based on datastore task data model.
-Task taskToDocument(datastore.Task task) {
```

Comment:
This is a function to convert a datastore Task to firestore Task, and it is still being used by multiple locations: https://github.com/search?q=repo%3Aflutter%2Fcocoon%20taskToDocument&type=code